packages/passport/sdk/src/zkEvm/zkEvmProvider.test.ts

```diff
@@ -101,6 +101,45 @@ describe('ZkEvmProvider', () => {
       });
     });
 
+    it('should emit accountsRequested event when the user is not authenticated', async () => {
```

Comment:
is there a specific event for 'session activity' that could be tested over this?

Reply:
I had the same thought that we might be able to test it a little bit more specifically. But I followed the test above which is similar to this. Lemme have a go though